SPECS/libvirt/libvirt.spec

```diff
@@ -1,654 +1,1970 @@
-Summary:        Network filter configuration files for the libvirtd daemon
+Summary: Network filter configuration files for the libvirtd daemon
 
-Requires:       libvirt-daemon = %{version}-%{release}
-Requires:       libvirt-daemon-driver-nwfilter = %{version}-%{release}
+Requires: libvirt-daemon-driver-nwfilter = %{version}-%{release}
 
 %description daemon-config-nwfilter
 Network filter configuration files for cleaning guest traffic
 
-%package daemon-driver-interface
-Summary:        Interface driver plugin for the libvirtd daemon
-
-Requires:       %{name}-daemon = %{version}-%{release}
-Requires:       %{name}-libs = %{version}-%{release}
-%if 0%{with netcf}
-Requires:       netcf-libs >= 0.2.2
-%endif
-
-%description daemon-driver-interface
-The interface driver plugin for the libvirtd daemon, providing
-an implementation of the host network interface APIs.
-
 %package daemon-driver-network
-Summary:        Network driver plugin for the libvirtd daemon
-
-Requires:       %{name}-daemon = %{version}-%{release}
-Requires:       %{name}-libs = %{version}-%{release}
-Requires:       dnsmasq >= 2.41
-Requires:       iptables
-Requires:       radvd
+Summary: Network driver plugin for the libvirtd daemon
+Requires: libvirt-daemon-common = %{version}-%{release}
+Requires: libvirt-libs = %{version}-%{release}
+Requires: dnsmasq >= 2.41
+Requires: iptables
 
 %description daemon-driver-network
 The network driver plugin for the libvirtd daemon, providing
 an implementation of the virtual network APIs using the Linux
 bridge capabilities.
 
-%package daemon-driver-nodedev
-Summary:        Nodedev driver plugin for the libvirtd daemon
-
-Requires:       %{name}-daemon = %{version}-%{release}
-Requires:       %{name}-libs = %{version}-%{release}
-Requires:       systemd
-
-%description daemon-driver-nodedev
-The nodedev driver plugin for the libvirtd daemon, providing
-an implementation of the node device APIs using the udev
-capabilities.
-
 %package daemon-driver-nwfilter
-Summary:        Nwfilter driver plugin for the libvirtd daemon
-
-Requires:       %{name}-daemon = %{version}-%{release}
-Requires:       %{name}-libs = %{version}-%{release}
-Requires:       ebtables
-Requires:       iptables
+Summary: Nwfilter driver plugin for the libvirtd daemon
+Requires: libvirt-daemon-common = %{version}-%{release}
+Requires: libvirt-libs = %{version}-%{release}
+Requires: iptables
+Requires: ebtables
 
 %description daemon-driver-nwfilter
 The nwfilter driver plugin for the libvirtd daemon, providing
 an implementation of the firewall APIs using the ebtables,
 iptables and ip6tables capabilities
 
-%package daemon-driver-qemu
-Summary:        QEMU driver plugin for the libvirtd daemon
+%package daemon-driver-nodedev
+Summary: Nodedev driver plugin for the libvirtd daemon
+Requires: libvirt-daemon-common = %{version}-%{release}
+Requires: libvirt-libs = %{version}-%{release}
+# needed for device enumeration
+Requires: systemd >= 185
+%if 0%{!?mariner}
```

Comment:
this is a new dependency that i don't see in mariner yet.  i commented it out for now.  not sure what the impact is of that, but the buddy build pipeline succeeds.